README.md

```diff
@@ -47,9 +47,9 @@ import Ember from 'ember';
 import { CanMixin } from 'ember-can';
 
 export default Ember.Route.extend(CanMixin, {
-  beforeModel: function() {
+  beforeModel() {
     if (!this.can('write post')) {
```

Comment:
good point, might make sense. the reason i bring all this up is that i often see issues where some mixin implements some global change, like page tracking or something, and that is not called when overriding these lifecycle events.